models/Student.js

```diff
@@ -0,0 +1,8 @@
+require('./StoryRead');
+module.exports = bookshelf.model('Student', {
+	tableName: 'students',
+	hasTimestamps: ['createdAt', 'updatedAt', 'deletedAt'],
+	authentication: function() {
```

Comment:
I think this should be changed from `authentication:...` to `storiesRead:`... or something like that